settings.gradle

```diff
@@ -1,4 +1,4 @@
 rootProject.name = 'terra-aws-resource-discovery'
 
 gradle.ext.projectGroup = 'bio.terra'
-gradle.ext.releaseVersion = 'v0.5.2'
+gradle.ext.releaseVersion = 'v0.6.0'
```

Comment:
there is a tag for this version but no release - 
https://github.com/verily-src/mc-terra-aws-wsm-landingzone/releases/tag/v0.6.0
is this intentional?

Reply:
Yes, we can't publish a release there until the avro changes here are merged, as the avro changes here need to propagate.